nginx-ubi/config/dhparams.pem

```diff
@@ -0,0 +1,8 @@
+-----BEGIN DH PARAMETERS-----
```

Comment:
this seems weird to hardcode this here - I'd expect this to be generated at deploy time (e.g. using `openssl dhparam -out dhparam.pem 4096`). should this be something the helm chart and/or operator is responsible for?

Reply:
It was weird to me to, but this is how we do it on our helm chart and in our container-dap repo as well.
What should we do?

Reply:
This is a pretty complex topic. See [here](https://conjurhq.slack.com/archives/CGTRB6G8G/p1595526917411100) for more info.

Reply:
It seems that in our helm chart we are using the same DH:
https://github.com/cyberark/conjur-oss-helm-chart/blob/master/conjur-oss/files/dhparams.pem
So what should be the resolution here?

Reply:
You can leave it as-is for now or generate on the fly. Generating a key larger than 1024 bits takes a very long time so just using this is better for now. The main important thing is that this file is generated by us and not a baked in-one from a distro but you can make your own with `openssl dhparam -out dhparams.pem 4096`. 

While testing the command, I also created one here too so feel free to use any of the 3 options:
```
-----BEGIN DH PARAMETERS-----
MIICCAKCAgEAlwtBWJfFjCcCFM+/7dS2oB1SwWWQcX7GMsqv4Gd1bFmzIb36ft43
gGFNURJQhplY0LnRWpTIPPChtGicWzTn267YnZ7dS/9g1HaF6Aqy1JZg1bwKY4fs
WOW11pY/W23p9G62DtvteoWmcVC8CF78NKm1PiDXdVc7z1MWH9GK7bJuzjZ7tlK7
Uwr9vOIuSFh/MPrhyW4EsMTEf0StWbLgb+UmfaAnAsExMbpbRNsmjpT/5r0I9UbK
KU287x3ED8T5Q4VfIGkDZFna1rxlZCVbOsYbRcMHXbsHAWYqLGMYgEmS/icXJpm2
RMzO8ne2Z36HqYmbLWCNblg8F2065QfJ9MvH0Y2muTrf4WXUTuVms5gA0ZtXc9hU
aIu8LOzEpJLa6V1/tU2PAGr9Aee9B/G5cV7kHVBjwbVFQMPA0FhjAKIqVI0L1IBa
rZ113AgTTRPpNH5g8A4exOFh6/mcMqI7dmdcfLtvzwHnJdXYDDE1JItdN6zCTWym
lzreYYBYYgyZpaY2n3LOnozyzYaRTGj3uLCvJS+EFRVIInc+8kzkYd7VWUK2+tve
w6W1L3Vx2LUiPWv1NM9disSoH4V2tU9Y7SD6qOM0kzeaQ3h1gIe0V26Uvq/MD1+t
PJHyKPmzAm8V56R2GOPYCldppi/xI3xfIHVHBnDcTWt2v2grkzvmw4sCAQI=
-----END DH PARAMETERS-----
```

Reply:
Please don't generate dhparams on the fly anywhere in CI. If you need readily accessible dhparams you can pull them out of ConjurOps from [here](https://github.com/conjurinc/conjurops-policies/blob/master/policy/dhparams.yml) (this is new and will be announced on Slack soon).

Reply:
@izgeri so its seems that we will keep it as is, could you please review the 2 checklist items and update how to proceed?